onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.h

```diff
@@ -115,6 +102,11 @@ class ModelBuilder {
   // the given node must be in the underlying graph_viewer
   const NodeUnit& GetNodeUnit(const Node* node) const;
 
+  const NnApi* GetNnApi() const { return nnapi_; }
+
+  const std::vector<ANeuralNetworksDevice*>& GetDevices() const { return nnapi_target_devices_; }
```

Comment:
do we need to add more accessors to the public interface? could GetNNAPIFeatureLevel() just take these values into account?

Reply:
Removed. It seems not necessary.